pkgs/top-level/all-packages.nix

```diff
@@ -9016,6 +9016,8 @@ in
 
   phantomjs2 = libsForQt5.callPackage ../development/tools/phantomjs2 { };
 
+  ping = callPackages ../development/web/ping { };
```

Comment:
```suggestion
  ping = callPackage ../development/web/ping { };
```

Reply:
Also I don't think this belongs in `development/web`.

This is a gui application that is useful for debugging web API's, so maybe `applications/networking`.

Reply:
I put it in ```development/web``` because ```postman``` was there, which is more or less the same application than ```ping```

Furthermore, it's a tool used for debugging API which are under development, so I think it makes sense to keep it here.

Reply:
Manual says that for things under `development` should be
> it’s used to support software development

And then goes on to mention libraries, compilers, and build-managers.
None on which will ever become a gui application.

You could say that an application like this would support software development, but so could an IDE, and they never should go under `development`.

I'd say that things like `postman` should be moved to a new place under `applications/` and the closest thing we have currently is something like `applications/networking`.

Reply:
I totally agree with you, I actually wanted to put the app under ```applications/networking``` before I discover that ```postman``` was under ```development/web```

What should I do then ? Keeping consistency with the actual organization or move ```ping``` under ```applications/networking``` to better agree with the manual ?

Reply:
I'd say `applications/networking` would be the best place currently.

We usually clean up the actual organization eventually.

Reply:
Alright, I will move it to the right place then!

Also, I was wondering, wouldn't it be more convenient to have the curated apps from Elementary App Center organized under ```desktop/pantheon``` in a special directory for them? Because some of these apps seem to be tightly coupled with some part of the desktop (e.g. Wingpanel, ...).

Reply:
> Alright, I will move it to the right place then!

Exclamation here confused me.

> Also, I was wondering, wouldn't it be more convenient to have the curated apps from Elementary App Center organized under desktop/pantheon in a special directory for them? Because some of these apps seem to be tightly coupled with some part of the desktop (e.g. Wingpanel, ...).

I thought of this too, and I've decided that apps that specifically won't function without some part of the pantheon desktop can be organized there.  
I've  also patched [granite](https://github.com/NixOS/nixpkgs/blob/32767d139f28fd3c00d687c04ec406258f7341e7/pkgs/desktops/pantheon/granite/default.nix#L15) so that they work outside pantheon (not needing wingpanel).
And even if they don't work I'd like them to default to not needing their desktop.
See: https://github.com/NixOS/nixpkgs/commit/ac2546bffe6c980642c9f6a71df06efa226b8816

Reply:
> Exclamation here confused me.

well, I was maybe a bit too enthusiastic that we found an agreement :smile:

I had no idea about granite, but it makes sense yes. And the idea of the flag to default to not needing their desktop seems convenient too. Thanks for the details.